content/en/dynamic_instrumentation/how-it-works.md

```diff
@@ -7,50 +7,53 @@ private: true
 
 ## Overview
 
-Dynamic instrumentation allows you to add probes to your production systems at any point in your code, even in third-party libraries. Dynamic Instrumentation has low overhead and is guaranteed to have no side effects on your system.
+Dynamic instrumentation allows you to add probes to your production systems at any location in your application's code, including third-party libraries. Dynamic Instrumentation has low overhead and is guaranteed to have no side effects on your system.
 
 ## Probe types
 
-You can create *snapshot probes* and *metric probes*.
+You can create *log probes* and *metric probes*.
 
-### Snapshot probes
+### Log probes
 
-Snapshot probes capture the context of a method or line, including:
+Log probes are enabled by default on all service instances that match the desired environment and version. They are rate limited to execute at most 5000 times per second, on each service instance.
 
-  - **Method arguments**, **local variables**, and **fields**, with the following limits:
-    - Three levels deep of class objects.
+If you enable `Capture method parameters and local variables` on the log probe, Dynamic Instrumentation captures the following data and adds it to the log event:
+  - **Method arguments**, **local variables**, and **fields**, with the following limits by default:
+    - Follow references three levels deep (configurable in the UI).
     - The first 100 items inside collections.
-    - 255 characters for string values.
-    - 20 collected fields in an object. Static fields are not collected.
+    - The first 255 characters for string values.
+    - 20 fields inside objects. Static fields are not collected.
   - Call **stack trace**.
   - Caught and uncaught **exceptions**.
 
-Snapshot probes are rate limited to one snapshot per second. 
+Since capturing extra data is a more performance intensive operation, by default it is only enabled on one instance of your service that matches the desired environment and version. Probes with snapshot capture are rate limited to execute once every second.
```

Comment:
```suggestion
Because capturing extra data is performance intensive, by default it is enabled on only one instance of your service that matches the specified environment and version. Probes with this capture setting enabled are rate limited to execute once per second.
```